docker-compose/cassandra-4.0/docker-compose-dev-mode.yml

```diff
@@ -17,7 +17,6 @@ services:
     environment:
       - JAVA_OPTS="-Xmx1536M"
       - CLUSTER_NAME=c4-${CASSTAG}-cluster
-      - CLUSTER_VERSION=4.0
```

Comment:
I think I'm missing something. Don't we need to give the PERSISTENCE_MODULE instead of the CLUSTER_VERSION?

Reply:
No, we don't need to provide it. The containers only have one persistence module, and I modified the Starter to handle that case.

Reply:
Got it!